reference/common/parameters/tidepoolfolloweduserid.v1.yaml

```diff
@@ -0,0 +1,7 @@
+description: >-
```

Comment:
[nit] To match `tidepoolsharerid` we might want to call this file `tidepoolfollowedid` (dropping the "user")?

Reply:
Actually, is this even used? I see the `followerUserId` is defined in `alerts.v1.yaml`.